src/cijoe/cli/cli.py

```diff
@@ -83,6 +84,28 @@ def cli_integrity_check(args):
     return 0
 
 
+def cli_script(args):
+    """Create python script with the boiler-plate code necessary for running cijoe as a standalone script."""
+    filename = "cijoe-script"
+    i = 0
```

Comment:
I think it would be preferable if this would always use the same filename, and then print an error and exit in case the script exists.

Reply:
E.g. something like:

```
with Path("cijoe-script.py") as path:
  if path.exists():
    output error-message
    return ....
    
   path.write_text(content)
```